src/Compute/Compute/Manual/PSVirtualMachinePatchAssessmentResult.cs

```diff
@@ -29,7 +29,7 @@ public class PSVirtualMachinePatchAssessmentResult
         public int? CriticalAndSecurityPatchCount { get; set; }
         public int? OtherPatchCount { get; set; }
         public DateTime? StartDateTime { get; set; }
-        public IList<VirtualMachineSoftwarePatchProperties> Patches { get; set; }
+        public IList<VirtualMachineSoftwarePatchProperties> AvailablePatches { get; private set; }
         public ApiError Error { get; set; }
```

Comment:
Please write a test for this as the initial functionality seems to have never been tested as this issue would have been caught back then.